packages/execute-driver/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@appium/execute-driver-plugin",
+  "version": "1.0.0",
+  "description": "Plugin for batching and executing driver commands with Appiums",
+  "main": "./build/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/appium/appium-plugins.git"
+  },
+  "appium": {
+    "pluginName": "execute-driver",
+    "mainClass": "ExecuteDriverPlugin"
+  },
+  "keywords": [
+    "appium",
+    "automation",
+    "webdriver"
+  ],
+  "author": "Appium <maintainers@appium.io>",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/appium/appium-plugins/issues"
+  },
+  "homepage": "https://github.com/appium/appium-plugins#readme",
+  "files": [
+    "build",
+    "docs",
+    "lib",
+    "index.js"
+  ],
+  "dependencies": {
+    "@appium/base-plugin": "^1.8.0",
```

Comment:
intentional caret?

Reply:
yeah i think it's left this way for monoerepo-internal deps for now, this should probably be updated to the `file:` style you mentioned doing in another monorepo cleanup PR.